tests/avocado/test_nec-usb-xhci.py

```diff
@@ -17,8 +18,12 @@ class TestNecUsbXhci(test.QemuTest):
     """
 
     def setUp(self):
+        # Getting latest Fedora Cloud Image
+        self.image = vmimage.get('Fedora')
```

Comment:
This is definitely an improvement.  Not having to create/copy or setup anyhow the image *by default* is very much a good thing.

Now, one further improvement would be to use defaults.  Ideally, this line and the following (`add_image()`) wouldn't be necessary on the test, unless the test writer wants to deviate from the default choices.

Reply:
You mean, every VM will have an image by default? I don't think that would be well received...